src/Microsoft.AspNet.Identity/SignInManager.cs

```diff
@@ -64,13 +64,13 @@ public class SignInManager<TUser> where TUser : class
         {
             if (Options.SignIn.RequireConfirmedEmail && !(await UserManager.IsEmailConfirmedAsync(user)))
             {
-                return await LogResultAsync(false, user);
+                return await false.WithLoggingAsync(this, user);
             }
             if (Options.SignIn.RequireConfirmedPhoneNumber && !(await UserManager.IsPhoneNumberConfirmedAsync(user)))
             {
-                return await LogResultAsync(false, user);
+                return await false.WithLoggingAsync(this, user);
```

Comment:
return await false.WithLoggingAsync just looks ugly to me, we are awaiting a bool?